src/cpp/redis.cpp

```diff
@@ -303,10 +303,17 @@ CommandReply Redis::run_model(const std::string& key,
                               std::vector<std::string> inputs,
                               std::vector<std::string> outputs)
 {
+    // Check for a non-default timeout setting
+    int run_timeout;
+    _init_integer_from_env(run_timeout, _MODEL_TIMEOUT_ENV_VAR,
+                           60 * 60 * 1000); // 1 hr
```

Comment:
We we define the default value as a constant like we do for other timeouts?  e.g.:

```c++
static constexpr int _DEFAULT_MODEL_TIMEOUT = 100;
```

Reply:
Good call; fixed